mopidy/config/__init__.py

```diff
@@ -182,6 +183,12 @@ def _validate(raw_config, schemas):
             errors[schema.name] = error
         if result:
             config[schema.name] = result
+        if schema.name in sections:
+            sections.remove(schema.name)
+
+    for section in sections:
+        logger.debug('Ignoring unknown config section: %s', section)
```

Comment:
Are you sure this shouldn't be at least at info level or maybe even on warning level?